resources/custom-elements-helpers.js

```diff
@@ -0,0 +1,262 @@
+function create_window_in_test(t, srcdoc) {
+  let p = new Promise((resolve) => {
+    let f = document.createElement('iframe');
+    f.srcdoc = srcdoc ? srcdoc : '';
+    f.onload = (event) => {
+      let w = f.contentWindow;
+      t.add_cleanup(() => f.parentNode && f.remove());
```

Comment:
Probably the issue is here; try not returning anything from the function argument.

Reply:
yes that was the issue

Reply:
OK, I'm not sure how add_cleanup interprets the return value of its function argument, but I trust Ms2ger here.

https://web-platform-tests.org/writing-tests/testharness-api.html#cleanup